.github/workflows/CI.yml

```diff
@@ -213,6 +213,59 @@ jobs:
         with:
           name: PDF build logs
           path: ${{ github.workspace }}/latex-debug-logs
+      - name: Upload search index
+        if: ${{ matrix.format == 'html' }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: search-index
+          path: docs/build/search_index.js
+
+  benchmarks:
+    name: Search Benchmarks
+    runs-on: ubuntu-latest
+    needs: docs
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
+        with:
+          version: 1
+          arch: x64
+          show-versioninfo: true
+      - uses: julia-actions/cache@v2
+      - uses: julia-actions/julia-buildpkg@v1
+      - name: Instantiate main project environment
+        run: julia --project=. -e 'using Pkg; Pkg.instantiate()'
```

Comment:
We need to instantiate the test environment here. There is [TestEnv](https://github.com/JuliaTesting/TestEnv.jl), but a better idea might be to use `[sources]` actually.

Reply:
https://github.com/JuliaDocs/Documenter.jl/pull/2740/commits/3f3200105c55dcf8b5b3f84b8c6be3d009afb081